cfpb/consumerfinance.gov
# TCCP: Tooltip accessibility improvements

Adds some code to close tooltips when `esc` key is pressed per [digital.gov guidance](https://designsystem.digital.gov/components/tooltip/accessibility-tests/). Also adds an `aria-label` to card results that summarizes all the card details to assistive technologies.

## How to test this PR

1. Everything should look the same.
1. Hover over a tooltip and click `esc` to close it.
1. `yarn cypress run --spec test/cypress/integration/consumer-tools/credit-cards/explore-cards.cy.js`

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

#### Browser testing

Check the current browser support cutoff list](https://cfpb.github.io/consumerfinance.gov/browser-support#current-browser-support-metrics) for browsers that are advisable
to prioritize for testing.

#### Accessibility


#### Other
